src/help/plugin.ts

```diff
@@ -149,19 +149,21 @@ function activate(app: JupyterLab, mainMenu: IMainMenu, palette: ICommandPalette
   // Handle state restoration.
   restorer.restore(tracker, {
     command,
-    args: widget => ({ isHidden: widget.isHidden, url: widget.url }),
-    name: widget => namespace
+    args: widget => ({ url: widget.url }),
```

Comment:
Shucks, this is because the restorer depends on the `disposed` signal of the widget.  The other widget's we've been using dispose of themselves when they are closed.  We can either:

- Make all iframes dispose on close
- Subclass iframe in this file to add this behavior
- Use a message hook for `close-request` messages in the tracker to remove the state